source/MaterialXRender/Handlers/ImageHandler.h

```diff
@@ -24,18 +24,32 @@ namespace MaterialX
 class ImageDesc
 {
   public:
+    enum class BaseType
```

Comment:
The class docstring above needs an update when images are no longer always float.